incubator/hnc/pkg/validators/hierarchy.go

```diff
@@ -141,6 +142,11 @@ func (v *Hierarchy) checkParent(ns, curParent, newParent *forest.Namespace) admi
 		return allow("parent unchanged")
 	}
 
+	if reconcilers.EX[newParent.Name()] {
```

Comment:
Do you mind moving EX out of `reconcilers` and into `pkg/config`? It doesn't make sense for the validators to import all the reconcilers just to get this one map, IMO.

Reply:
Absolutely! I left in default_config.go as it seemed most appropriate. I suppose if we wanted to expand on the exclusion concept we can find another home for it.